runtime/browser/android/xwalk_web_contents_delegate.cc

```diff
@@ -264,8 +264,9 @@ bool XWalkWebContentsDelegate::ShouldCreateWebContents(
   if (java_delegate.obj()) {
     ScopedJavaLocalRef<jstring> url_buffer =
         base::android::ConvertUTF8ToJavaString(env, target_url.spec());
-    return Java_XWalkWebContentsDelegate_shouldOpenWithDefaultBrowser(env,
-        java_delegate.obj(), url_buffer.obj()) == JNI_FALSE;
+    // Always return false for android runtime
+    Java_XWalkWebContentsDelegate_shouldOpenWithDefaultBrowser(env,
+        java_delegate.obj(), url_buffer.obj());
```

Comment:
It's useful to check the result of shouldOpenWithDefaultBrowser. This method may cause to launch the default browser and open the link. If this happened, we should not allow the XWalkView to create another window.

Besides, multiple windows mode has already been implemented by onCreateWindowRequested(XWALK-2374) and released in embedding API docs. But if ShouldCreateWebContents always return false, it will disable this published API.